lib/dest/write-contents/write-symbolic-link.js

```diff
@@ -1,18 +1,44 @@
 'use strict';
 
-var fs = require('graceful-fs');
+var path = require('path');
 
 var fo = require('../../file-operations');
 
 function writeSymbolicLink(file, optResolver, onWritten) {
-  // TODO handle symlinks properly
-  fs.symlink(file.symlink, file.path, function(symlinkErr) {
-    if (fo.isFatalOverwriteError(symlinkErr)) {
-      return onWritten(symlinkErr);
-    }
-
-    onWritten();
-  });
+  var srcPath = file.symlink;
+
+  var isDirectory = file.isDirectory();
```

Comment:
Told you it was ugly :-P

The sync call was just to make it clear, it'd be async of course and only ever arise in a few unlikely cases on an already crappy OS.

I appreciate and share your impatience for release... But imho it would then be better to rip out junctions completely rather than having it half-baked -- especially for such a highly anticipated release?

By the way, I think I might have mentioned earlier: has anyone ever checked if node doesn't do this testing for directory targets for us, if it's in fact true that junctions don't work or make sense for anything other than directories? What happens if we pass 'junction' in an `fs.symlink` call if the target is a regular file?